app/models/timdex_wrapper.rb

```diff
@@ -0,0 +1,57 @@
+class TimdexWrapper
+  def initialize
+    @timdex_http = HTTP.persistent(timdex_url)
```

Comment:
I'm not sure how long you intend the TimdexWrapper to stay around after initialization. If it's for a single query, I think we should remove the persistent connection.

If you have in mind a way to initialize once and re-use the same instance of TimdexWrapper across multiple searches the persistent connection is a value for sure.

If you aren't sure yet we can talk it through and decide which will make the most sense based on what we know so far.

Reply:
Honestly, I grabbed this approach from Bento but didn't think critically about it. Looking over documentation now.